tutorials/monitoring/tutorial.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Monitoring NServiceBus solutions
+reviewed: 2017-10-02
+summary: Getting started with the monitoring capabilities built in to the Particular Service.
+---
+
+This tutorial will guide you through getting started with the monitoring capabilities built in to the Particular Service Platform.
+
+You can follow this tutorial to configure any NServiceBus based solution for monitoring with the Particular Service Platform. If you do not have a solution to monitor just yet, you can get started by downloading our sample solution.
+
+downloadbutton
+
+
+## Component overview
+
+The following diagram shows all the components involved in monitoring an NServiceBus solution with the Particular Service Platform.
+
+```mermaid
+graph TD
+
+subgraph Endpoint1
+  metrics1(NServiceBus.Metrics) --> plugin1
+  plugin1(NServiceBus.Metrics.ServiceControl)
+end
+
+subgraph Endpoint2
+  metrics2(NServiceBus.Metrics) --> plugin2
+  plugin2(NServiceBus.Metrics.ServiceControl) 
+end
+
+plugin1 --> q(Monitoring Queue)
+plugin2 --> q(Monitoring Queue)
+
+q --> scmonitoring[Monitoring Instance]
+
+scmonitoring --> pulse[ServicePulse]
+```
+
+![the above renders like this](component-overview.png)
+
+Each endpoint in the solution is configured to collect metrics about it's own operation using the NServiceBus.Metrics NuGet package. This package hooks into the NServiceBus pipeline to calculate and expose key metric data. It also adds headers to outgoing messages to track queue length.
```

Comment:
![image](https://user-images.githubusercontent.com/677704/31216034-4f44351a-a9b2-11e7-82f4-b46dc9419fd3.png)

It could just be the "mysterious" term "key metric data" that is bothering me. I'd probably feel more comfortable if it read something like:

> This package hooks into the NServiceBus pipeline to calculate and expose metrics will can tell us about the health and performance of an endpoint.